api/envoy/extensions/filters/network/redis_proxy/v3/redis_proxy.proto

```diff
@@ -363,6 +363,10 @@ message RedisProxy {
   // If this setting is set together with ``downstream_auth_username`` and ``downstream_auth_password``, the external auth service will be source of truth, but those fields will still be used for downstream authentication to the cluster.
   // The API is defined by :ref:`RedisProxyExternalAuthRequest <envoy_v3_api_msg_service.redis_auth.v3.RedisProxyExternalAuthRequest>`.
   RedisExternalAuthProvider external_auth_provider = 10;
+
+  // A List of custom command names already defined on Redis other than the already supported commands,
+// the list of commands mentioned below will get passed on to Redis.
+  repeated string redis_custom_command_names = 11;
```

Comment:
I don't know much about how Redis commands work, but if the list is predefined, should this be an enum instead of an arbitrary string?  I see the code makes a check to see if the name is a valid custom command - if we keep this a string, can we note what the system behavior is if an invalid command is specified in this list?

Reply:
So redis allows use to load modules which can define any kind of commands like redis has a SET command but you can define something like SET_A that takes only one param and sets the key A (just an example there would be better practical use cases.)

As a result we can't keep this as an enum, I'll test and document the behaviour in case the command specified doesn't exist.

Reply:
Got it.  Are there any concerns around arbitrary strings being a vector for an injection attack on Redis?